frontend/src/hooks/useCollaborateData.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useLayoutEffect, useEffect } from 'react';
+import { SWRConfiguration, SWRResponse } from 'swr';
+import { dequal } from 'dequal';
+import { StaleDataNotification } from 'component/common/StaleDataNotification/StaleDataNotification';
+
+interface IFormatUnleashGetterOutput<Type> {
+    data: Type;
+    refetch: () => void;
+}
+
+const formatUnleashGetter = <Type,>({
+    unleashGetter,
+    dataKey = '',
+    refetchFunctionKey = '',
+    options = {},
+    params = [''],
+}: IGetterOptions): IFormatUnleashGetterOutput<Type> => {
+    const result = unleashGetter(...params, { ...options, refreshInterval: 5 });
```

Comment:
Would it make sense to have refreshInterval be part of the options passed down from the hook? That way we could have the flexibility to set it on a case-by-case scenario, while still having the default of 5.

Reply:
Yeah, it should be in the reverse order here. I'll fix that, thanks!